contrib/platform/src/com/sun/jna/platform/win32/Wininet.java

```diff
@@ -0,0 +1,372 @@
+
+import com.sun.jna.Native;
+import com.sun.jna.Pointer;
+import com.sun.jna.Structure;
+import com.sun.jna.Union;
+import com.sun.jna.platform.win32.WinNT.HANDLE;
+import com.sun.jna.ptr.IntByReference;
+import com.sun.jna.win32.StdCallLibrary;
+import com.sun.jna.win32.W32APIOptions;
+
+/**
+ * Functions in WinInet.dll
+ */
+public interface Wininet extends StdCallLibrary {
+    /**
+     * A usable instance of this interface
+     */
+    Wininet                 INSTANCE                  = (Wininet) Native.loadLibrary("wininet", Wininet.class,
+            W32APIOptions.DEFAULT_OPTIONS);
+
+    /**
+     * Normal cache entry; can be deleted to recover space for new entries.
+     */
+    public static final int NORMAL_CACHE_ENTRY        = 1;
+
+    /**
+     * Sticky cache entry that is exempt from scavenging for the amount of time
+     * specified by dwExemptDelta.<br>
+     * The default value set by CommitUrlCacheEntryA and CommitUrlCacheEntryW is
+     * one day.
+     */
+    public static final int STICKY_CACHE_ENTRY        = 4;
+
+    /**
+     * Cache entry file that has been edited externally. This cache entry type
+     * is exempt from scavenging.
+     */
+    public static final int EDITED_CACHE_ENTRY        = 8;
+
+    /**
+     * Not currently implemented.
+     */
+    public static final int TRACK_OFFLINE_CACHE_ENTRY = 16;
+
+    /**
+     * Not currently implemented.
+     */
+    public static final int TRACK_ONLINE_CACHE_ENTRY  = 32;
+
+    /**
+     * Partial response cache entry.
+     */
+    public static final int SPARSE_CACHE_ENTRY        = 65536;
+
+    /**
+     * Cookie cache entry.
+     */
+    public static final int COOKIE_CACHE_ENTRY        = 1048576;
+
+    /**
+     * Visited link cache entry.
+     */
+    public static final int URLHISTORY_CACHE_ENTRY    = 2097152;
+
+    /**
+     * Closes the specified cache enumeration handle.
+     * 
+     * @param hFind
+     *            Handle returned by a previous call to the
+     *            FindFirstUrlCacheEntry function.
+     * @return Returns TRUE if successful, or FALSE otherwise. To get extended
+     *         error information, call GetLastError.
+     */
+    public boolean FindCloseUrlCache(HANDLE hFind);
+
+    /**
+     * @param lpszUrlName
+     *            String that contains the name of the source that corresponds
+     *            to the cache entry.
+     * @return Returns TRUE if successful, or FALSE otherwise.<br>
+     *         To get extended error information, call GetLastError.<br>
+     *         Possible error values include the following.<br>
+     *         <ul>
+     *         <li><b>ERROR_ACCESS_DENIED:</b>The file is locked or in use. The
+     *         entry is marked and deleted when the file is unlocked.</li>
+     *         <li><b>ERROR_FILE_NOT_FOUND:</b>The file is not in the cache.
+     *         </li>
+     *         </ul>
+     */
+    public boolean DeleteUrlCacheEntry(String lpszUrlName);
```

Comment:
No need for _public_ - all methods in an interface are such - please remove the modifier in all the methods